packages/blocks/src/_common/components/block-caption.ts

```diff
@@ -100,12 +100,12 @@ export class BlockCaptionEditor<
   override connectedCallback(): void {
     super.connectedCallback();
 
-    this.caption = this.block.model.caption;
+    this.caption = this.model.caption;
 
     this.disposables.add(
-      this.block.model.propsUpdated.on(({ key }) => {
+      this.model.propsUpdated.on(({ key }) => {
         if (key === 'caption') {
-          this.caption = this.block.model.caption;
+          this.caption = this.model.caption;
           if (!this._focus) {
             this.display = !!this.caption?.length;
           }
```

Comment:
May be this can rewrite with signal based props

Reply:
You're right. But it would be better open a new PR to do this sort of things.